### DISCUSSION TITLE
为啥label都“混乱”了

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#2064] [提问]
这label好让人捉摸不透啊，`拒绝/放弃`竟然跟新反馈一个色儿，重复与忽略改色了，甚至还有什么推迟等待重要，咋搞出来的啊……
~这个等待真有意思~
其实我知道这些label只是为了让划分更加正确细致，但是为什么要改呢，我只想知道这个

**Comment by Chiloven945** [comment]
更好区分啊……推迟等待重要因该是之前的αβγ演变过来的（？

**Comment by Deep-Dark-Forest** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/131975043/a2832d3d-0410-4c00-a207-13f46f5137c6)
没事，我色盲（

**Reply by iPodtouch6557** [reply]
阿这，色盲模式？

**Reply by Deep-Dark-Forest** [reply]
> 阿这，色盲模式？

是“高对比度模式”啦（
